packages/core/src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -40,6 +44,15 @@ export const Main: StoryObj<HvDatePickerProps> = {
   argTypes: {
     classes: { control: { disable: true } },
   },
+  parameters: {
+    eyes: {
+      runBefore() {
+        fireEvent.click(screen.getByRole("combobox"));
+
+        return waitFor(() => screen.getByRole("tooltip"));
```

Comment:
Don't we need to `await` the element to be on the page? 🤔
```suggestion
        await waitFor(() => screen.getByRole("tooltip"));
```

Reply:
From what I've read [here](https://www.npmjs.com/package/@applitools/eyes-storybook#runbefore-and-runafter-functions) it's not necessary. The `runBefore` accepts promises and resolves them before taking the screenshot.